index.html

```diff
@@ -12,6 +12,66 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    <!--Navbar-->
+    <div class="container">
+        <div class="logo">
+          <img src="/img/karma-logo.svg" alt="logo">
+          </div>
+        <nav>
+          <ul>
+          <li><b>Meet Karma</b></li>
+          <li>How it Works</li>
+          <li>Store</li>
+          <li>Blog</li>
+          <li>Help</li>
+          <li>Login</li>
+          </ul>
+        </nav>
+    </div>
+
+        <div class="bckimg">
+            <br>
+            <br>
+            <br>
+            <br>
```

Comment:
Don't rely on HTML to create space on your page. This is the job of CSS. What type of spacing can you create to add room inside `bckimg`?